app/models/distribution.rb

```diff
@@ -125,4 +127,14 @@ def future?
   def past?
     issued_at < Time.zone.today
   end
+
+  def issued_at_end_is_after_issued_at
+    if issued_at_timeframe_enabled
+      start_time = issued_at
+      end_time = issued_at_end
+      if start_time > end_time
```

Comment:
```suggestion
      if issued_at > issued_at_end
```
You can directly access these values and then use them for the conditional. However, we can keep it this way if the variables `start_time` and `end_time` make things clearer.